styles/css/vtex.product-summary.css

```
@@ -36,3 +36,7 @@
 .imageContainer {
   text-align: center;
 }
+
+.image {
+  border-radius: 4px;
```

I don't know if my suggestion is better, but I think that using `rem` would make it closer with the [tachyons token](https://twitter.com/RafaelKlynger/status/1246931223915749382)
```suggestion
  border-radius: 0.25rem;
```

---

But `rem` is relative to the screen right? I don't think that we want to change the border if the screen changes, but I can check this out

---

Done!